src/utils/optimizely.js

```diff
@@ -0,0 +1,65 @@
+export const EVENTS = {
+  ENROLLMENT_CLICK: 'enterprise_learner_portal_enrollment_click',
+}
+
+export const EXPERIMENT_0 = {
+  ID: 21468970242,
+  VARIANT_IDS: {
+    CONTROL: 21446530045,
+    VARIANT_1: 21458980337,
```

Comment:
> // double-equals because it's unclear to me if the variant ids returned by optimizely are always strings

if optimizely seems to be returning these values as strings, should the data type in this object reflect that? it feels like doing extra work to convert the values to a `Number` here and in `getActiveExperiments` below? 🤔

Reply:
[curious] Should these be hardcoded ID's or environment variables, or...?

Reply:
1. Good point, I'll make these data objects use string ids.
2. Good question!  If we could have nested config outside of code, they could be outside of code.  I think it's ok to leave them here because Optimizely-related code is transient by its nature (which is a grown-up way of saying "we're messily hacking with reckless abandon").

Reply:
Agreed on point 2 :) Just wanted to clarify.

Reply:
One thing I'm ignoring here, though, is that the experiment ids will be different per environment (because they're different experiments).  So I think I'll have to move this data into environment variables like:
```
EXPERIMENT_0_ID=1234
EXPERIMENT_0_VARIANT_1=6789
EXPERIMENT_1_ID=9999
EXPERIMENT_1_VARIANT_1=0000
EXPERIMENT_1_VARIANT_2=1111
```